Makefile

```diff
@@ -1,30 +1,39 @@
-OSFLAG:=$(shell go env GOHOSTOS)
-BINARY:=ipxe
-IPXE_BUILD_SCRIPT:=binary/script/build_ipxe.sh
-IPXE_NIX_SHELL:=binary/script/shell.nix
+OSFLAG := $(shell go env GOHOSTOS)
+BINARY := ipxe
+IPXE_BUILD_SCRIPT := binary/script/build_ipxe.sh
+IPXE_FETCH_SCRIPT := binary/script/fetch_and_extract_ipxe.sh
+IPXE_NIX_SHELL := binary/script/shell.nix
 
 help: ## show this help message
 	@grep -E '^[a-zA-Z_-]+.*:.*?## .*$$' Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}'
 
 include lint.mk
 
+.PHONY: binary
```

Comment:
sorry, just caught this now. Is this wanted? `make binary` is running every time for me.

Reply:
we do want this because it is a phony target, it should be ok because nothing calls `make binary` unless we know we need to rebuild. Its rebuilding for you because `make binary` isn't succeeding. I'm not sure why its not working for you can you post the full make output? 

"Works on my machine"
```
[16:49:55]-[~/g/s/g/t/ipxedust]-[manny@zennix]
git clean -fxd; make binary &>/dev/null; echo status=$status
status=0
[16:50:32]-[~/g/s/g/t/ipxedust]-[manny@zennix]
make binary
make: Nothing to be done for 'binary'.
```

Reply:
ahh...ok. that make sense, thanks. Let me do another run and see.

Reply:
Maybe i had a different commit because it's working now. Thanks!